docs/content/docs/reference/react/hooks.mdx

```diff
@@ -0,0 +1,393 @@
+```
+
+## User management
+
+### Change email
+
+```js
+const { change, isLoading, isSuccess, needsVerification, isError, error } =
+  useChangeEmail(email?: string, options?: { redirectTo?: string })
+```
+
+- `changeEmail: (email?: string) => void` requests the email change. The arguement password will take precedence over the the possible state value used when creating the hook.
+- `isLoading: boolean` returns `true` when the action is executing, `false` when the action result is available.
+- `needsVerification: boolean` returns `true` if the email change has been requested, but that a email has been sent to the user to verify the new email.
+- `isError: boolean` returns `true` if an error occurred.
+- `error?: {status: number, error: string, message: string}` provides details about the error.
+- `redirectTo?: string`: redirection path in the client application that will be used in the link in the verification email. For instance, if you want to redirect to `https://myapp.com/success`, the `redirectTo` value is `'/success'`.
+
+#### Usage
+
+```jsx
+import { useState } from 'react'
+import { useChangeEmail } from '@nhost/react'
+
+const Component = () => {
+  const [email, setEmail] = useState('')
+  const { changeEmail, isLoading, needsVerification, isError, error } = useChangeEmail(password)
+  return (
+    <div>
+      <input value={email} onChange={(event) => setEmail(event.target.value)} />
+      <button onClick={changeEmail}>Change password</button>
+      {needsVerification && (
+        <div>
+          Please check your mailbox and follow the verification link to confirm your new email
+        </div>
+      )}
+    </div>
+  )
+}
+```
+
+### Change password
+
+```js
+const { changePassword, isLoading, isSuccess, isError, error } = useChangePassword(password?: string)
+```
+
+- `changePassword: (password?: string) => void` requests the password change. The arguement password will take precedence over the the possible state value used when creating the hook.
+- `isLoading: boolean` returns `true` when the action is executing, `false` when the action result is available.
+- `isSuccess: boolean` returns `true` if the password has beed successfully changed.
+- `isError: boolean` returns `true` if an error occurred.
+- `error?: {status: number, error: string, message: string}` provides details about the error.
+
+#### Usage
+
+```jsx
+import { useState } from 'react'
+import { useChangePassword } from '@nhost/react'
+
+const Component = () => {
+  const [password, setPassword] = useState('')
+  const { changePassword, isLoading, isSuccess, isError, error } = useChangePassword(password)
+  return (
+    <div>
+      <input value={password} onChange={(event) => setPassword(event.target.value)} />
+      <button onClick={changePassword}>Change password</button>
+    </div>
+  )
+}
+```
+
+### Reset password
+
+When an user lost their password, we can resend them an email to authenticate, so they can change their password:
```

Comment:
"If a user loses their password, we can resend them an email to authenticate so that they can change their password:"